server-side/controllers/profile.controller.js

```diff
@@ -1,77 +1,81 @@
-import mongoose  from 'mongoose';
-import Profiles from '../models/profile.model.js';
+import Profile from '../models/profile.model.js';
 
-
-export const getAllProfiles = async (req, res, next) => {
+export const getAllProfiles = async (req, res) => {
     try {
-        const profiles = await Profiles.find().populate('limitedWebsites.websiteId blockedSites').select('-__v');
+        const profiles = await Profile.find();
```

Comment:
you shouldn't fetch from the collection from the collection, you should do it from service file - do it in all your code